src/main/scala/com/fulcrumgenomics/bam/EstimateRnaSeqInsertSize.scala

```diff
@@ -312,6 +312,19 @@ object EstimateRnaSeqInsertSize {
   }
 }
 
+/**
+  * Metrics produced by `EstimateRnaSeqInsertSize` to describe the distribution of insert sizes within an
+  * RNA-seq experiment.
```

Comment:
I know I wrote this metric, so thank-you for adding docs.  How about adding a note that the insert size is computed using the transcripts as the reference, so in "transcript space"?